keyboards/hexboard/rgb_matrix_kb.inc

```diff
@@ -0,0 +1,16 @@
+//rgb_matrix_mode(RGB_MATRIX_CUSTOM_my_off_effect);
+
+RGB_MATRIX_EFFECT(my_off_effect)
+
+#ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
+
+// A simple effect for LEDs to start off, without affecting the global hsv brightness
```

Comment:
you could also set the default to "off", too.  setting `rgb_matrix.default.on` to false should do that, and without the custom animation needed

Reply:
That doesn't seem to do what I want (though maybe because the keyboard remembers the previous effect that was used?)

Reply:
I just removed the custom effect. It's fine without it.